lib/internal/module.js

```diff
@@ -44,3 +48,27 @@ function stripBOM(content) {
   }
   return content;
 }
+
+exports.builtinLibs = ['assert', 'buffer', 'child_process', 'cluster',
+  'crypto', 'dgram', 'dns', 'domain', 'events', 'fs', 'http', 'https', 'net',
+  'os', 'path', 'punycode', 'querystring', 'readline', 'stream',
+  'string_decoder', 'tls', 'tty', 'url', 'util', 'v8', 'vm', 'zlib'];
```

Comment:
> I guess it doesn't matter too much.

Agreed. And I just noticed that the list misses the `repl` module itself? Should I add that myself here? :smile: